build/Makefile

```diff
@@ -382,19 +396,32 @@ push-controller-image: $(ensure-build-image)
 	docker push $(controller_tag)
 
 # build the static binary for the gameserver sidecar
-build-agones-sdk-binary: $(ensure-build-image) build-agones-sdk-binary-linux build-agones-sdk-binary-windows build-agones-sdk-binary-darwin
+ifeq ($(WITH_ARM64), 1)
```

Comment:
So for the SDK binary, I say let's always build all the sidecar binaries.

Since we have https://agones.dev/site/docs/guides/client-sdks/local/ - we might as well have them all, since we should ship a zip of all them with each release anyway.

So I think we can remove the `WITH_ARM64` condition on `build-agones-sdk-binary`.